app/src/main/res/layout/fragment_evaluation_result.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This layout is not really "screen size change friendly" but you can leave it like this, I will do a full cleanup of the layouts at the end of the project.

Reply:
sounds good, thanks